src/main/java/uk/gov/hmcts/sscs/service/CcdService.java

```diff
@@ -122,10 +123,16 @@ public CaseData findCcdCaseByAppealNumberAndSurname(String appealNumber, String
         return caseData.getAppeal() != null && caseData.getAppeal().getAppellant() != null
                 && caseData.getAppeal().getAppellant().getName() != null
                 && caseData.getAppeal().getAppellant().getName().getLastName() != null
-                && caseData.getAppeal().getAppellant().getName().getLastName().equalsIgnoreCase(surname)
+                && compareSurnames(surname, caseData)
                 ? caseData : null;
     }
 
+    private boolean compareSurnames(String surname, CaseData caseData) {
+        String caseDataSurname = caseData.getAppeal().getAppellant().getName().getLastName().replaceAll("[^a-zA-Z]", "");
+        String unidecodeSurname = unidecode(surname).replaceAll("[^a-zA-Z]", "");
+        return unidecode(caseDataSurname).equalsIgnoreCase(unidecodeSurname);
```

Comment:
Shouldn't we also remove non alpha characters from the case data surname, to get a match?